app/src/main/java/com/rlab/sejima/features/MUHeader.java

```diff
@@ -0,0 +1,138 @@
+package com.rlab.sejima.features;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Color;
+import android.graphics.Typeface;
+import android.text.TextUtils;
+import android.util.AttributeSet;
+import android.util.DisplayMetrics;
+import android.util.TypedValue;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.rlab.sejima.R;
+
+public class MUHeader extends RelativeLayout {
+
+    /**
+     * The default size of title
+     */
+    private final static int DEFAULT_TITLE_SIZE_IN_SP = 34;
+    /**
+     * The default size of detail
+     */
+    private final static int DEFAULT_DETAIL_SIZE_IN_SP = 14;
+
+    /**
+     * The current title
+     */
+    public String mTitle = "";
+    /**
+     * The title's size
+     */
+    public float mTitleSize;
```

Comment:
mTitleSize and mDetailSize are the only variables without default value. Any reason to don't use 34 and 14 directly instead of using static private var?

Reply:
It’s all about easy-reading.
To set a font size for a TextView, values have to be given in pixels (px) whereas 14 and 34 are both density-pixels sizes (dp). I used static values (`DEFAULT_TITLE_SIZE_IN_SP` and `DEFAULT_DETAIL_SIZE_IN_SP`) to make the code more easy-reading. 
Most of time, when you specify a font-size in the XML file of the view, you give a dp size and when you retrieve it in the view Class file, it’s automatically converted in px, so I can’t affect directly 14 and 34 without converting them before, that’s why I set `mTitleSize` and `mDetailSize` to 0. 
Finally, when I apply the font-sizes, I check these values. If they’re not null, then they are directly given in pixels, otherwise, I have to convert my static values (`DEFAULT_TITLE_SIZE_IN_SP` and `DEFAULT_DETAIL_SIZE_IN_SP`).
It enables to keep the default value in dp.